public/secure-inbox.html

```
@@ -0,0 +1,194 @@
+        <i class="fas fa-lock text-teal-600 dark:text-teal-400"></i> Secure Inbox
+        <span id="inbox-count" class="text-sm font-normal text-gray-500"></span>
+      </h1>
+      <p class="text-sm text-gray-500 mt-1">Messages expire 7 days after delivery. Download to keep a copy.</p>
+    </div>
+  </section>
+
+  <main class="flex-1 max-w-3xl w-full mx-auto px-4 py-6 space-y-6">
+    <section class="rounded-xl border border-gray-200 dark:border-gray-700 p-4">
+      <h2 class="text-lg font-semibold mb-3">Compose Secure Message</h2>
+      <form id="compose-form" class="space-y-3">
+        <input id="recipient" type="text" placeholder="Recipient username" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)